### DISCUSSION TITLE
Please open issues in the repo

### DISCUSSION THREAD

**Discussion by baseplate-admin** [pelican-themes/papyrus#8] [Ideas]
Hi, It would be awesome if you folks opened issues in this repository. Thanks :D

**Comment by aluminum-ice** [comment]
I do not see an option to open an issue in the repo? Do you have it turned on? I just updated my fork to roll in your recent changes and I see a small issue. When in dark mode, clicked links don't appear because the color scheme is dark blue it seems against a black background. Have you noticed this? See the two screenshots below.

![Screenshot 2023-11-30 at 3 58 34 PM](https://github.com/pelican-themes/papyrus/assets/12374267/c4974592-8015-45e8-b49c-10111e702106)

![Screenshot 2023-11-30 at 3 58 28 PM](https://github.com/pelican-themes/papyrus/assets/12374267/a82ce574-ed00-4c61-8aa4-6b458a1cc33c)
